source/config/configSpec.py

```diff
@@ -112,6 +112,11 @@
 	width = integer()
 	height = integer()
 	displays = string_list()
+	fontPointSize = integer()

```

Comment:
I think I would like these options below to have defaults defined. See may comment below about catching exceptions when fetching config parameters to set the font